fvm/environment/account_info.go

```diff
@@ -254,3 +268,20 @@ func (info *accountInfo) GetAccount(
 
 	return account, nil
 }
+
+func (info *accountInfo) GetAccountKeys(
+	address flow.Address,
+) (
+	[]flow.AccountPublicKey,
+	error,
+) {
+	defer info.tracer.StartChildSpan(trace.FVMEnvGetAccount).End()
```

Comment:
can you add a new trace constant `FVMEnvGetAccountKeys` here
https://github.com/onflow/flow-go/blob/dfd15534a32ce07085fc41759bbb256d3fd1abce/module/trace/constants.go#L164